### PR TITLE
* Add SetValue to PropertyHelper

### PR DESCRIPTION
**Pull request by pranavkm**
* Expose created getter and setters as properties on PropertyHelper.

### CODE REVIEW

File: src/Microsoft.Framework.PropertyHelper.Internal/PropertyHelper.cs [hunk 1]
```diff
@@ -48,12 +49,44 @@ public PropertyHelper([NotNull] PropertyInfo property)
 
         public virtual string Name { get; protected set; }
 
+        /// <summary>
+        /// Gets the property value getter.
+        /// </summary>
+        public Func<object, object> ValueGetter => _valueGetter;
```
Comment (harshgMSFT) [inline src/Microsoft.Framework.PropertyHelper.Internal/PropertyHelper.cs]: aaarrgh!  .. :smile:

File: test/Microsoft.Framework.Internal.Test/PropertyHelperTest.cs [hunk 2]
```diff
@@ -42,6 +42,54 @@ public void PropertyHelper_ReturnsValueCorrectly()
...
+        {
+            // Arrange
+            var expected = "new value";
+            var instance = new BaseClass { PropA = "old value" };
+            var helper = PropertyHelper.GetProperties(
+                instance.GetType()).First(prop => prop.Name == "PropA");
+
+            // Act and Assert
+            Assert.NotNull(helper.ValueSetter);
+            helper.ValueSetter(instance, expected);
```
Comment (harshgMSFT) [inline test/Microsoft.Framework.Internal.Test/PropertyHelperTest.cs]: Change the name of the test  you are not calling SetValue.

**Comment by harshgMSFT** [comment]
:shipit:

File: src/Microsoft.Framework.PropertyHelper.Internal/PropertyHelper.cs (hunk 1 above)
Comment (rynowak) [inline src/Microsoft.Framework.PropertyHelper.Internal/PropertyHelper.cs]: 31 #PranavPoints awarded.

File: src/Microsoft.Framework.PropertyHelper.Internal/PropertyHelper.cs [hunk 3]
```diff
@@ -48,12 +49,44 @@ public PropertyHelper([NotNull] PropertyInfo property)
...
+                    _valueSetter = MakeFastPropertySetter(Property);
+                }
+
+                return _valueSetter;
+            }
+        }
+
         public object GetValue(object instance)
         {
             return _valueGetter(instance);
```
Comment (rynowak) [inline src/Microsoft.Framework.PropertyHelper.Internal/PropertyHelper.cs]: Might as well use the new property you added

**Comment by rynowak** [comment]
:shipit: from me after fixing minor stuff

**Pull request closed without merging**